client/src/components/messages/Messages.tsx

```diff
@@ -3,26 +3,53 @@ import axios from 'axios';
 import ConversationList from './ConversationList';
 import ConversationView from './ConversationView';
 import io from 'socket.io-client';
+import Chip from '@mui/material/Chip';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
 
 const socket = io('http://localhost:4000');
 
-interface Conversation {
+type Conversation = {
   id: number;
+  label: string;
+  participants: { username: string }[];
+}
+
+type User = {
+  id: number;
+  name: string;
+  username: string;
+  googleId: string;
+  linkedinId: string;
+  githubId: string;
+  picture: string;
+  firstName: string;
+  lastName: string;
+  follower_count: number;
+  post_count: number;
 }
```

Comment:
This can be imported from `import User from '@prisma/client' so it will stay up to date with the current schema

Reply:
Ah didn't know that, I'll make the change.